libs/form/src/next/input-field-masked/vl-input-field-masked.component.ts

```diff
@@ -126,18 +119,33 @@ export class VlInputFieldMaskedComponent extends VlInputFieldComponent {
             const customTransformFn = this.maskOptions?.customTransformFn;
             const transformedValue = customTransformFn ? customTransformFn(value) : value;
 
-            if (transformedValue !== value) {
-                this.value = transformedValue;
+            if (this.value === transformedValue) {
+                // Request een update zodat de waarde van de input teruggezet wordt naar de vorige waarde.
                 this.requestUpdate();
                 return;
             }
 
-            this.value = value;
-            this.dispatchEvent(
-                new CustomEvent('vl-input', { composed: true, bubbles: true, detail: { value: this.value } })
-            );
+            this.value = transformedValue;
         }, 0);
     }
+
+    protected onUpdated(changedProperties: Map<string, unknown>) {
```

Comment:
Als ik me niet vergis, moet dit `private` zijn ipv `protected`? Idem waar dit hergebruikt wordt.

Reply:
Zoals besproken, is `protected` omdat `vl-input-field-masked` die overschrijft dus dit is normaal > resolved